### PR TITLE
style: prettier format stylesheets

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

- [X] the [contributor license agreement][] is signed
- [X] commit message follows [commit guidelines][]
- [ ] tests are included
- [ ] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change

1. Adds css and scss files to format, check, and lint-staged 9cea6c1
2. Cleans up invalid css and scss syntax 35092dd
3. Apply `npm run format:scss` 58da9b8 

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: Source/Plugins/Core/com.equella.core/resources/web/css/equella-theme.css [hunk 1]
```diff
@@ -1,63 +1,75 @@
...
 #tasks-pager span,
 #pager a,
 #pager span {
-	color: #DDD;
+  color: #ddd;
 }
 #pager .k-state-active,
 #notes-pager .k-state-active {
-	background color: #222;
+  background-color: #222;
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/resources/web/css/equella-theme.css:26]: `background color` is not a css rule `background-color` is

File: Source/Plugins/Core/com.equella.core/resources/web/css/share.rtl.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 @import url("share.css");
 
 .shareButton {
-	url(../images/share-action.rtl.png) right 0 no-repeat
-}
\ No newline at end of file
+  background: url(../images/share-action.rtl.png) right 0 no-repeat;
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/resources/web/css/share.rtl.css:4]: `url` needs to be applied to a rule

File: Source/Plugins/Core/com.equella.core/resources/web/css/share/share.rtl.css [hunk 3]
```diff
@@ -1,5 +1,5 @@
 @import url("share.css");
 
 .shareButton {
-	url(../images/share-action.rtl.png) right 0 no-repeat
-}
\ No newline at end of file
+  background: url(../images/share-action.rtl.png) right 0 no-repeat;
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/resources/web/css/share/share.rtl.css:4]: `url` needs to be applied to a rule

File: package.json [hunk 4]
```diff
@@ -5,11 +5,13 @@
     "postinstall": "cross-env-shell \"./coursier bootstrap com.geirsson:scalafmt-cli_2.12:1.5.1 -f -o node_modules/.bin/scalafmt && ./coursier bootstrap com.google.googlejavaformat:google-java-format:1.7 -f -o node_modules/.bin/google-java-format\"",
     "format:scala": "find . -name \"*.scala\" -o -name \"*.sbt\" | xargs scalafmt",
     "format:java": "find . -name \"*.java\" | xargs google-java-format -r",
+    "format:scss": "prettier --write \"Source/Plugins/Core/com.equella.core/{js,resources}/**/*.{css,scss}\"",
     "format:ts": "eslint --fix \"Source/Plugins/Core/com.equella.core/js/**/*.{js,ts,tsx}\"",
     "check:scala": "find . -name '*.scala' -o -name '*.sbt' | xargs scalafmt --test",
     "check:java": "find . -name '*.java' | xargs google-java-format -n --set-exit-if-changed",
     "check:license": "cd Source/Plugins/Core/com.equella.core/js && licensee --production --errors-only && cd -",
     "check:md": "remark -q -f -u remark-validate-links -u remark-lint-no-dead-urls --ignore-pattern NOTICE.md .",
+    "check:scss": "prettier --check \"Source/Plugins/Core/com.equella.core/{js,resources}/**/*.{css,scss}\"",
```
Comment (edalex-ian) [inline package.json:14]: Being a longer line I guess, this has highlighted to me the pattern of duplication we have in here of lines which are definitely prone to typo errors. Like we've done in the [Core package.json](https://github.com/openequella/openEQUELLA/blob/develop/Source/Plugins/Core/com.equella.core/js/package.json#L4), should we start using the [package.json config object](https://docs.npmjs.com/files/package.json#config)?

So here we could add:

```json
"config": {
  "stylesheet_glob": "\"Source/Plugins/Core/com.equella.core/{js,resources}/**/*.{css,scss}\""
}
```

And then we simply have:

```json
"format:scss": "prettier --write ${npm_package_config_stylesheet_glob}",
.
.
.
"check:scss": "prettier --check ${npm_package_config_stylesheet_glob}",
```

And later we can come back and address the others. (Or if you're willing, I see now harm in updating the others now as part of this.)

Comment (ChristianMurphy) [reply package.json]: sure, added in 55505f0 for stylesheets and typescript

**Comment by ChristianMurphy** [comment]
> Would like us to start parametrising the duplication in package.json though, so if you could start with this change that'd be cool.

parameterized in 55505f0

> Fixing some of the errors could result in some visual impacts for users. But should be super minor, so hopefully a non-issue.

We can fix them or remove them.
I'm okay either way, whichever you think will have the best outcome for the project.
We may not want to leave them broken though :sweat_smile:

**Comment by edalex-ian** [comment]
I reckon good to go. :+1:

**Pull request merged**